f34-py39/s2i_assemble.patch

```diff
@@ -83,6 +86,11 @@
 -  pip install -r requirements.txt
 -fi
 +thamos install
```

Comment:
```suggestion
+thamos install -- --disable-pip-version-check --no-cache-dir
```

thamos uses pip under the hood, it can pass any options to pip as necessary:

https://github.com/thoth-station/thamos/#installing-requirements

When using environment variables (see bellow), the options would be automatically propagated.